lib/datadog/di/component.rb

```diff
@@ -55,12 +55,12 @@ def environment_supported?(settings, logger)
           # TODO add tests?
           unless settings.dynamic_instrumentation.internal.development
             if Datadog::Core::Environment::Execution.development?
-              logger.debug("Not enabling dynamic instrumentation because we are in development environment")
+              logger.warn("datadog: di: not enabling dynamic instrumentation because we are in development environment")
```

Comment:
Minor: Perhaps this is more of an info thing?

Reply:
It's a minor detail, but my thinking is -- will it be more likely that someone enabled DI because they really want to add extra instrumentation on something they're running on their laptop OR that they enabled it unconditionally on their config as it's simpler to do that and don't really want/need DI in development?

I'm thinking the latter sounds a bit more likely? But no numbers to back it up haha :)